src/bcs/VacuumConcBC.C

```diff
@@ -7,22 +7,37 @@ VacuumConcBC::validParams()
 {
   InputParameters params = IntegratedBC::validParams();
   params += ScalarTransportBase::validParams();
+  MooseEnum vacuum_bc_type("marshak mark milne", "marshak");
```

Comment:
Should these be uppercase (to match with VaccumConcBC.h) or should they stay lowercase (to match NtAction.C?)